### ISSUE TITLE
Black screen when changing levels with use emulator help to r/w frame buffers

### ISSUE THREAD

**Issue by CadetSparklez** [gonetz/GLideN64#2582]
I noticed this in dk64 when loading from the menu, changing worlds through a portal (either level or boss) and when exiting huts? such as cranky/funky but not when entering. This also occurs in rogue squadron not when you finish a level, but after you leave the results/medal screen)
(audio still plays but no inputs are accepted either)

At first I noticed changing to lle mode fixed it, so then I switch on and off certain options and narrowed it down to this one, I didn't test in rs yet but I assume its the same setting there as well. (not even sure if lle mode in m64p is glides lle mode or it just switches to a different plugin.

It would be nice if this could get fixed (if it is fixable) since when uncapping game speed I noticed significantly better performance (~30%) with it turned on in rs, even if I did stupid things like read the colour buffer in 4k chunks and didn't read the depth buffer in 4k chunks.

Thanks!

**Comment by Rosalie241** [comment]
> not even sure if lle mode in m64p is glides lle mode or it just switches to a different plugin.

m64p doesn't even ship with GLideN64 anymore, but on the releases it did, LLE mode switched it to parallel-rdp and parallel-rsp.

**Comment by CadetSparklez** [comment]
Yeah, I prefer glideN64 for a number of reasons so I use an old update of m64p, thx for the heads on on lle mode

**Comment by gonetz** [comment]
"use emulator help to r/w frame buffers" is potentially dangerous option. This functionality is poorly supported by emu core.
I'd better use frame buffer read in sync mode, if hardware allows to to do it fullspeed.

> noticed significantly better performance (~30%) with it turned on in rs

What is rs?

**Comment by CadetSparklez** [comment]
> What is rs?
oh rogue squadron, not rainbow six or racing simulation 2, sorry

**Comment by Xii-Nyth** [comment]
> > not even sure if lle mode in m64p is glides lle mode or it just switches to a different plugin.
> 
> m64p doesn't even ship with GLideN64 anymore, but on the releases it did, LLE mode switched it to parallel-rdp and parallel-rsp.

Ok so after losing my save state (m64p had the eeprom size set wrong for dk64 (knew it, just couldn't change it easily like in 1964) until a newer update which came out after gliden64 builds stopped releasing) so I grabbed a good save off the internet but it black screened when loading in after the zipper on the banana.

So I went and got your mupen gui which I recently found out existed when editing the gametechwiki page after being annoyed that there wasn't texture back support in m64p or depth output in the retroarch proprietary one, because retroarch itself has no depth output even though glide does. (why'd it have to be a proprietary core.. it was the chosen one)

Anyways that didn't black screen in the same spot, so I went to the nearest portal to test this out again and it just doesn't have the problem! Dunno if that was you or a change in the plugin but the others are stuck on an old version of the plugin so it was your effort either way that made it work.

imo not having netplay support is the only thing keeping all the other emulators alive compared to your amazing one, besides gepd for its game specific stuff.

Can you start a patreon? I'll give you 20$ a month to keep up the good work which I don't know how to do myself
